utils/mysql.cnf

```diff
@@ -0,0 +1,60 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This is a pre-configuration for Docker MySQL to allow BinLog and replication necessary for Debezium
+# see ./openmrs-compose.yaml
+
```

Comment:
nit: If the original template for this file is copied from somewhere, consider adding a reference here.

Reply:
Noted, Thanks!